Packs/Pcysys/Integrations/Pcysys/Pcysys.py

```diff
@@ -187,9 +187,10 @@ def _parse_date(full_date, separator):
 
         res_list: List[dict] = []
         for ordered_dict in full_action_report_list:
-            full_date_to_convert = ordered_dict['Time'].replace(' ', '')
-            full_date_list = full_date_to_convert.split(',')
-            day, month, year = _parse_date(full_date_list[0], '/')
+            full_date_to_convert = ordered_dict['Time']
+            LOG(f'full_date_to_convert: {full_date_to_convert}')
+            full_date_list = full_date_to_convert.split(' ')
+            year, month, day = _parse_date(full_date_list[0], '-')
```

Comment:
maybe you would like to use library like dateutil so the format will not hurt you as much in the future?

Reply:
For sure, it is only a temporary fix for this version. In the next release I'll wrap it up with a library